engine/classes/Elgg/Config.php

```diff
@@ -289,6 +292,77 @@ public function loadSettingsFile() {
 	}
 
 	/**
+	 * Pre 3.0 certain config values were store in the database
+	 * To allow for a smooth upgrade, we will attempt to copy these settings from the database
+	 * to settngs.php, where these values will be read from by 3.0 codebase
+	 *
+	 * @return void
+	 */
+	public function migrateDbSettings($path) {
```

Comment:
Ideally, we should make these easy to implement. Perhaps some directly in `/install` where we execute them all.